src/controllers/userController.js

```
@@ -0,0 +1,132 @@
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const User = require('../models/userModel');
+const Movie = require('../models/movieModel');
+
+const {
+    getUsers,
+    getUsersById,
+    addUser,
+    updateUser,
+    removeUser
+} = require('../services/userService');
+
+const { statusCode } = require('../helpers/constants');
+
+const getAllUsers = async (req, res, next) => {
+    const users = await getUsers();
+    res.status(statusCode.OK).json(users);
+}
+
+const getUser = async (req, res, next) => {
+    const { id } = req.params;
+    const user = await getUsersById(id);
+    if (!user) {
```

You use asyncWrapper in routers, so you dont need next fucntion in controller handlers. You can throw an error from the service, or controller without using next function